src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import AddAssignment from './AddAssignment';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
```

Comment:
I will not merge for now. There are changes to the base files (which are required but not this way). Try to avoid changes to the base files for pull requests.